integration/spark-common-cluster-test/src/test/scala/org/apache/spark/sql/common/util/QueryTest.scala

```diff
@@ -220,4 +218,74 @@ object QueryTest {
 
     return None
   }
+
+  /**
+   * execute the query by establishing the jdbc connection
+   *
+   * @param query
+   * @return
+   */
+  def executeQuery(dbName: String, query: String): List[Map[String, Any]] = {
+    Try {
+      val conn: Connection = if (System.getProperty("presto.jdbc.url") != null) {
```

Comment:
I think it is not a good idea to create connection for each query

Reply:
yes, optimized it. 
same problem was there in presto UT also. Fixed it.